requirements-build.txt

```diff
@@ -1,7 +1,7 @@
 alembic
 apispec
 apispec-webframeworks
-bokeh
+bokeh>=3.0.0
```

Comment:
I think this PR's changes would still work pre-3.0, but it'll be nice to floor it anyway.